test/utils.spec.js

```diff
@@ -0,0 +1,7 @@
+import utils from '../src/utils'
```

Comment:
Without the default export in utils, you could use
```suggestion
import { arrayUnique } from '../src/utils'
```

Reply:
Yes, I were waiting for Caleb's confirmation. Thanks!